opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -689,16 +711,24 @@ def get_a0( self, t=None, iteration=None, pol=None ):
         """
         if pol not in ['x', 'y']:
             raise ValueError('The `pol` argument is missing or erroneous.')
-
         if pol == 'x':
-            slicing_dir = 'y'
             theta = 0
         else:
-            slicing_dir = 'x'
             theta = np.pi / 2.
+        if "3dcartesian" in self.avail_geom:
+            slicing = 0.
+            if pol == 'x':
+                slicing_dir = 'y'
+            else:
+                slicing_dir = 'x'
+        else:
+            slicing = None
```

Comment:
Why is it important to set `slicing` in the above code?  Shouldn't the default `slicing` give pretty much the same result?

Reply:
Right, this no longer make sense with these default values.